common/src/main/java/org/apache/gravitino/credential/CredentialPropertyUtils.java

```diff
@@ -53,7 +54,9 @@ public class CredentialPropertyUtils {
           OSSTokenCredential.GRAVITINO_OSS_SESSION_ACCESS_KEY_ID,
           ICEBERG_OSS_ACCESS_KEY_ID,
           OSSTokenCredential.GRAVITINO_OSS_SESSION_SECRET_ACCESS_KEY,
-          ICEBERG_OSS_ACCESS_KEY_SECRET);
+          ICEBERG_OSS_ACCESS_KEY_SECRET,
+          ADLSTokenCredential.GRAVITINO_ADLS_SAS_TOKEN,
+          ICEBERG_ADLS_TOKEN);
```

Comment:
The iceberg token name is not correct, it should be `adls.sas-token.{account_name}`

Reply:
correct, the token name for Iceberg shoud be `adls.sas-token.{account_name}.dfs.core.windows.net`